backend/src/directors/chat_director.py

```diff
@@ -20,6 +21,11 @@
 engine = PromptEngine()
 director_prompt = engine.load_prompt("chat_director")
 
+@dataclass
+class FinalAnswer:
+    message: Optional[str] = ""
```

Comment:
I made a mistake with this suggestion, this should be `message: str = ""`. The Optional is saying that message can be type `str` or `None`, however, we are defaulting this to empty string so saying it can be `None` doesn't make sense